iron-input.html

```diff
@@ -103,6 +103,25 @@
       'keypress': '_onKeypress'
     },
 
+    registered: function() {
+      if (/Firefox|Trident/.test(navigator.userAgent)) {
```

Comment:
In theory we can do feature detection to figure out if we need to do this. I think that we should prefer feature detection (where possible, and within reason). I'm going to circle around with @kevinpschaaf  to make sure there aren't any hidden gotchas that would prevent this... but, do you know of any?